test/extended/image_ecosystem/plugin.go

```diff
@@ -378,6 +378,19 @@ func getAdminPassword(oc *exutil.CLI) string {
 	return "password"
 }
 
+// Finds the pod running Jenkins
+func findJenkinsPod(oc *exutil.CLI) *kapi.Pod {
+	pods, err := oc.KubeREST().Pods(oc.Namespace()).List(kapi.ListOptions{})
+	o.ExpectWithOffset(1, err).NotTo(o.HaveOccurred())
+	for _, p := range pods.Items {
+		if strings.HasPrefix(p.Name, "jenkins") && !strings.Contains(p.Name, "deploy") && p.Status.Phase == "Running" {
+			return &p
+		}
+	}
+	g.Fail("Should have detected jenkins pod", 1)
+	return nil
+}
+
 func followDCLogs(oc *exutil.CLI, jenkinsNamespace string) {
```

Comment:
Yeah I could see this method having usefulness beyond our jenkins tests ... perhaps a candidate as well for moving into test/extended/util as well, with the DC name being a param as well.